db/seeds.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
-User.create!(email: 'test@test.com', password: 'password')
+User.create!(email: 'test@test.com', password: 'password',
```

Comment:
@mrkava 

For example, your teammate will add something to seed.rb, and you will need it for your ticket. Then you will run seed second time )